client/components/table/empty.js

```diff
@@ -0,0 +1,33 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * `EmptyTable` displays a blank space with an optional message passed as a children node
+ * with the purpose of replacing a table with no rows.
+ * It mimics the same height a table would have according to the `numberOfRows` prop.
+ *
+ * @return { object } -
+ */
+const EmptyTable = ( { children, numberOfRows } ) => {
+	return (
+		<div className="woocommerce-table is-empty" style={ { '--number-of-rows': numberOfRows } }>
```

Comment:
TIL about inline variables, cool!

Reply:
I didn't know about them either until I had to use them here! :slightly_smiling_face: